Microsoft.Data.Analysis.Interactive/DataFrameKernelExtension.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
```

Comment:
Can you add a copyright to this file? You can find one at the top of any existing file in the Microsoft.Data.Analysis library.

Reply:
sure